tests/integration/targets/mso_schema/tasks/main.yml

```diff
@@ -115,3 +122,131 @@
     that:
     - query_non_existing is not changed
     - query_non_existing.current == {}
+
+# TEST BLOCK FOR MSO VERSION >= 4.1 WHICH ALLOW FOR SCHEMA WITHOUT TEMPLATES
+- name: Execute tasks only for MSO version >= 4.1
+  when: version.current.version is version('4.1', '>=')
+  block:
+
+    - name: Ensure empty schemas are not present
+      cisco.mso.mso_schema:
+        <<: *mso_info
+        schema: '{{ item }}'
+        state: absent
+      loop:
+      - "empty_schema"
+      - "empty_schema_changed"
+
```

Comment:
is it possible to change the schema names to ansible_test and ansible_test_2 instead of empty_schema and empty_schema_changed, as this will help us clean up the resources.

Reply:
changed